src/components/Header/Header.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components'
+import HeaderLogo from '../../components/assets/Logo/HeaderLogo.svg'
+
+export default function Header() {
+  return (
+    <>
+      <StyledHeader>
+        <img src={HeaderLogo} alt="Logo for the App Habits" />
+      </StyledHeader>
+    </>
+  )
+}
+
+const StyledHeader = styled.div`
+  top: 0;
+  z-index: 100;
+  position: fixed;
```

Comment:
`position: fixed;` takes your component out if the normal document flow, which is why your form is overwritten by your header. Since it is your HTML element, it will automatically be ontop of your page. You leave out the first three css properties and it should still work the same.